docs/3.0rc/develop/write-tasks.mdx

```diff
@@ -701,31 +664,4 @@ These settings will not override the `retries` or `retry_delay_seconds` that are
 ```
 prefect config set PREFECT_TASK_DEFAULT_RETRIES=2
 prefect config set PREFECT_TASK_DEFAULT_RETRY_DELAY_SECONDS = [1, 10, 100]
-```
-
-## Task results
```

Comment:
I think this section is too advanced for this page and better covered in this section: https://docs-3.prefect.io/3.0rc/develop/task-runners#configure-a-task-runner